lib/puppeteer_environment.js

```diff
@@ -0,0 +1,33 @@
+const chalk = require('chalk')
+const NodeEnvironment = require('jest-environment-node')
+const puppeteer = require('puppeteer')
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+const DIR = path.join(os.tmpdir(), 'jest_puppeteer_global_setup')
+
+class PuppeteerEnvironment extends NodeEnvironment {
+  async setup () {
+    console.log(chalk.yellow('Setup Test Environment.'))
```

Comment:
Are these comments still useful, or should they be removed now?

Reply:
I thought would be good to know what happens and when.
![screen shot 2018-03-08 at 15 08 58](https://user-images.githubusercontent.com/788096/37158174-a8c1432c-22e2-11e8-8d39-dd4103491ace.png)
Can remove them all.

Reply:
Happy to keep them in if they are useful :)